HCSP.js

```diff
@@ -0,0 +1,228 @@
+{
+    "translatorID": "3e375db7-5f6e-4861-ab8b-3ddb0f4accc9",
+    "label": "HCSP",
+    "creator": "Joris Muller",
+    "target": "http://www.hcsp.fr",
+    "minVersion": "2.1",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "g",
+    "lastUpdated": "2015-08-04 14:29:59"
+}
+
+/*
+HCSP Translator
+A Zotero's Translator for the website of "Haut Conseil de Santé Publique"
+Copyright (C) 2015 Joris Muller <jomuller@jom.link>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Usage of Zotero.Utilities.xpath() function. Exist only since Zotero 2.1
+// https://www.zotero.org/support/dev/client_coding/changes_in_zotero_2.1#xpath_utility_functions
+// Then minimal version of Zotero needed is 2.1
+
+// TODO: Use camelStyle for variables to be consistent with Zotero coding style
+
+function detectWeb(doc, url) {
+    /* The URLs have upper or lower case on the website
+    depending the way you access to a page */
+    if (url.match("[Ee]xplore.cgi/avisrapportsdomaine")) {
+        return 'report';
+    } else if (url.match("[Ee]xplore.cgi/avisrapports")) {
+        return 'multiple';
+    }
+}
+
+function doWeb(doc, url) {
+
+    // Store articles URLs in a array
+    var articles = [];
+
+    if (detectWeb(doc, url) == "multiple") {
+
+        // Titles and URLs of each item are on the third column of the only table
+        // This column is the only one with links (<a>)
+        var xpath_titles = '//tbody/tr/td/a';
+        var titles = ZU.xpath(doc, xpath_titles);
+        
+        // Put all the titles and their URLs in the items object
+        // NOTE: "for of loop" are quite new and works only with newest browsers
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
+        var allItems = {};
+        for (var nextTitle of titles) {
+            allItems[nextTitle.href] = nextTitle.textContent;
+        }
+        
+        // Let the user choose the items he wants
+        var itemsSelected;
+        itemsSelected = Zotero.selectItems(allItems);
```

Comment:
```Zotero.selectItems``` should be run as a callback function. See e.g. https://github.com/zotero/translators/blob/d82c1efd2ca75af0caa0d94884e38d2e6afbc210/Dryad%20Digital%20Repository.js#L60 
for an example that otherwise does the same you do.